README.md

```diff
@@ -4,14 +4,16 @@ This is a [Stytch](https://stytch.com) + [Next.js](https://nextjs.org/) project
 
 In this repo, we have two sample auth flows:
 
-- SDK integration: This flow uses Stytch's React component to create a login and signup flow using email magic links. Other libraries used include Stytch's node client library for authentication and with-iron-sessions for session management.
-- API integration: This flow uses a custom UI with Stytch's backend API for SMS OTP authentication. Both Stytch's node client library and with-iron-session are also used.
+- SDK integration: This flow uses Stytch's React component to create a login and sign-up flow using [Email Magic Links](https://stytch.com/docs/api/send-by-email).
+- API integration: This flow uses a custom UI with Stytch's backend API for [Onetime Passcodes(OTP) via SMS](https://stytch.com/docs/api/sms-otp-overview) authentication.
 
-# Getting Started
+Both flows use Stytch's [Node client library](https://github.com/stytchauth/stytch-node) and `with-iron-session` for session management.
```

Comment:
I didn't know what `with-iron-session` was. A quick search gave me [`iron-session`](https://www.npmjs.com/package/iron-session). Is that the same thing? Is it worth making this a link?

Reply:
The intention is to update to use Session Management soon, but this is a good call out and will hyperlink it in the meantime.

Added in 970004b.